src/main/java/net/glowstone/entity/objects/GlowMinecart.java

```diff
@@ -27,8 +29,17 @@
 import org.bukkit.material.MaterialData;
 import org.bukkit.util.Vector;
 
+// TODO: Minecarts stop when there's an arrow in the way.
 public abstract class GlowMinecart extends GlowEntity implements Minecart {
 
+    @Getter @Setter private volatile double damage;
```

Comment:
Could the annotations be on separate lines please? Also, why are these volatile?

Reply:
They're volatile because we can't guarantee they'll be accessed from only one thread.